remappers/HumanDynamicsRemapper/conf/HumanDynamicsRemapperExample.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+<robot name="iCub-Retargeting" build=0 portprefix="">
```

Comment:
I think this `iCub-Retargeting` name was copied from another file, you could change it with something more appropriate such as `Human-Dynamics-Remapper`